src/include/daos/cont_props.h

```diff
@@ -137,4 +137,10 @@ daos_cont_prop2ec_pda(daos_prop_t *prop);
 uint32_t
 daos_cont_prop2rp_pda(daos_prop_t *prop);
 
+static inline uint32_t
+daos_cont_props2pda(struct cont_props *props, bool is_ec_obj)
+{
+	return is_ec_obj ? props->dcp_ec_pda : props->dcp_rp_pda;
```

Comment:
sorry, what is the difference between ec_pda and rp_pda?

Reply:
it is just for different type of obj class, details may refer HLD.
just paste related info here -
For a large EC object, avoiding one or few objects consume all bandwidth of a performance domain is the main concern, so spreading shards to many groups can avoid bandwidth hogging and benefit overall performance. However, for an object with a small redundancy group size (e.g., three-way replicated object), bandwidth consumption is already limited by shards (targets) number. Instead of spreading shards to different performance domains, placing shards of this type of object in the same performance domain is the optimal option because it has a few benefits:
• Avoid extra network-hops of forwarding RPCs between servers and reduce latency of regular I/O.
• Reduce data transfer between DAOS groups during rebuild.